README.md

```diff
@@ -1,35 +1,54 @@
-# Commands to run the server
-
-* 1) Run database locally
-    mongod or sudo mongod
-    * In case it's not installed, install mongo: https://docs.mongodb.com/v3.2/administration/install-community/
-
-
-* 2) Install server dependencies:
+# Commands to install and run the backend application
+
+* Download and install node version 12.18.0 https://nodejs.org
```

Comment:
Let's not be that specific, 12.18.0.. Let's keep it as 12.x.x

Reply:
this would be updated